JASP-Desktop/analysis/analysisform.h

```diff
@@ -52,19 +52,24 @@ class AnalysisForm : public QQuickItem, public VariableInfoProvider
 	Q_PROPERTY(QQuickItem * errorMessagesItem	READ errorMessagesItem	WRITE setErrorMessagesItem	NOTIFY errorMessagesItemChanged	)
 	Q_PROPERTY(bool			needsRefresh		READ needsRefresh									NOTIFY needsRefreshChanged		)
 	Q_PROPERTY(bool			hasVolatileNotes	READ hasVolatileNotes								NOTIFY hasVolatileNotesChanged	)
+	Q_PROPERTY(bool			runAnalysisWhenOptionChanged	READ runAnalysisWhenOptionChanged	WRITE setRunAnalysisWhenOptionChanged NOTIFY runAnalysisWhenOptionChangedChanged )
```

Comment:
Wouldn't `runOnChange` be a little bit more succint?
I mean, it is obviously about an analysis, because it is a form and that it is an option should also be obvious.

Reply:
Done